index.html

```diff
@@ -1,98 +1,165 @@
-                    in favor of the new Dualsense controller. The new controller come equipped with haptic feedback on the controller, a new touchpad, and much more ergonomic style that mimics the larger Xbox controller. With console generations lasting roughly between 6-7 years, the PS5 has a few years for developers
-                    to grow more accustomed to the hardware and take advantage of all the PS5 has to offer.</p>
-            </div>
-            <div> 
-                <p>
-                    <a href="https://www.statista.com/statistics/1101872/unit-sales-video-game-consoles/">Console Sale Projections</a>
-                    <br>
-                    <a href="https://www.lifewire.com/playstation-3-all-you-need-to-know-2717718">What is the Playstation 3?</a>
-                    <br>
-                    <a href="https://www.playstationlifestyle.net/2020/08/14/report-ps5-cooling-solution-liquid-metal-patent/">PS5 Liquid Cooling</a>
-                    <br>
-                    <a href="https://www.britannica.com/topic/PlayStation">Brief Playstation History</a>
-                </p>
-            </div>
-        </main>  
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>History of Playstation's Consoles</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <script src="./js/script.js" defer></script>
+  </head>
+  <body>
+    <nav>
+      <a href="#playstation-one">PS</a>
+      <a href="#playstation-two">PS2</a>
+      <a href="#playstation-three">PS3</a>
+      <a href="#playstation-four">PS4</a>
+      <a href="#playstation-five">PS5</a>
+    </nav>
+    <header>
+      <h1>History of Playstation's Console</h1>
+    </header>
+    <main>
+      <section id="playstation-one">
+        <h2>Playstation</h2>
+        <div class="fancy-video">
+          <video src="./videos/playstation-one.mp4" muted></video>
+          <div class="replay">
+            <img src="./images/refresh.svg" alt="Replay" />
+          </div>
+        </div>
+        <p>
+          Originally a joint product with Nintendo, Sony released the
+          Playstation in 1994.This was the first console in history to ship over
+          100 million units under 10 years. First branded by Sony executive Ken
+          Kutaragi, the design was meant to be an extension of the "Super
+          Famicom" and dubebd the "SNES-CD." When negotiations ended, Sony
+          Interactive greenlit the project. The console shipped with
+          firsty-party exclusive games like
+          <i>Rayman</i>,<i>Crash Bandicoot</i>, and <i>Gran Turismo.</i>
+        </p>
+      </section>
```

Comment:
I reworked the nesting of your elements here. Previously, everything was inside of the `fancy-video` wrapper. This is what a messing up your autoplaying video. It made the element unnecessarily tall, such that it would not trigger a `intersection`.

It makes sense to wrap up related content, so I moved the heading and the paragraph outside of `fancy-video` but then wrapped them all in a `section`.